src/components/Books/CreateNewBook.js

```diff
@@ -13,19 +33,27 @@ const CreateNewBook = () => {
           type="text"
           placeholder="Book title"
           className="grid-item"
+          name="title"
+          value={inputBook.title}
+          onChange={onChange}
+          required
         />
```

Comment:
- [ ] The form for adding the book should have two inputs, one for the `title` and another for the `author`. Please kindly fix that 🙃 🙃

Reply:
Fixed, thank you.